tests/scripts/booking/cancel_old_unused_bookings_for_venue_test.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime, timedelta
+import pytest
+from pcapi.core.bookings.models import Booking
+
+from pcapi.model_creators.generic_creators import create_booking, create_deposit, create_offerer, create_user, create_venue
+from pcapi.repository import repository
+from pcapi.scripts.booking.cancel_old_unused_bookings_for_venue import cancel_old_unused_bookings_for_venue
+from pcapi.utils.human_ids import humanize
+
+
+@pytest.mark.usefixtures("db_session")
+def test_should_cancel_old_unused_bookings_for_venue():
+    venue_id = 10
+    offerer = create_offerer()
+    venue = create_venue(idx=venue_id, offerer=offerer)
+    beneficiary = create_user()
+    create_deposit(user=beneficiary)
+
+    to_cancel_booking_id = 11
+    used_booking_id = 12
+    recent_booking_id = 13
+
+    to_cancel_booking = create_booking(date_created=(
+        datetime.now() - timedelta(days=40)), user=beneficiary, venue=venue, idx=to_cancel_booking_id)
+
+    used_booking = create_booking(date_created=(
+        datetime.now() - timedelta(days=40)), user=beneficiary, venue=venue, idx=used_booking_id, is_used=True)
+
+    recent_booking = create_booking(date_created=(
+        datetime.now() - timedelta(days=10)), user=beneficiary, venue=venue, idx=recent_booking_id)
+
+    repository.save(to_cancel_booking, used_booking, recent_booking)
+
+    cancel_old_unused_bookings_for_venue(
+        humanize(venue_id))
+
+    to_cancel_booking_result: Booking = Booking.query.filter(
+        Booking.id == to_cancel_booking_id).first()
+    used_booking_result: Booking = Booking.query.filter(
+        Booking.id == used_booking_id).first()
+    recent_booking_result: Booking = Booking.query.filter(
+        Booking.id == recent_booking_id).first()
```

Comment:
Je ne suis pas sûr que le typing soit vraiment utile ici. D'autre part, on peut utiliser `get()` pour faire plus concis :

```suggestion
    to_cancel = Booking.query.get(to_cancel.id)
    used_booking = Booking.query.get(used_booking.id)
    recent_booking = Booking.query.get(recent_booking.id)
```

Reply:
c'est notamment pour avoir de l'auto-complétion pour + bas mais ok